lib/utils.js

```diff
@@ -56,15 +56,16 @@ function stripHttp1ConnectionHeaders (headers) {
 
 // issue ref: https://github.com/fastify/fast-proxy/issues/42
 function buildURL (source, reqBase) {
+  let baseOrigin = reqBase ? new URL(reqBase).href : undefined
```

Comment:
this is using `.href` instead of `.origin` btw, otherwise a bunch of the tests verifying this throws started failing

Reply:
Ok! I'm trusting the tests - do you think there is the possibility of a bug there?

Reply:
Nah, I don't think so. This makes sure the user provides cleaner inputs without "patching" over what is provided which seems reasonable 🙂